src/wwwroot/styles/views/_header.scss

```diff
@@ -102,7 +102,6 @@ header{
       ul{
         list-style-type: none;
         color: $suit-font-color-lighter;
-        text-transform: uppercase;
```

Comment:
Sorry, yes I checked, in the translation is lowercase 
![2017-07-20_1002](https://user-images.githubusercontent.com/6796523/28418607-abc49ca6-6d32-11e7-8ee6-b359c53b1734.png)